tests/baselines/reference/tscWatch/programUpdates/when-changing-`checkJs`-of-config-file.js

```diff
@@ -0,0 +1,170 @@
+currentDirectory:: /user/username/projects/myproject useCaseSensitiveFileNames: false
+Input::

```

Comment:
Should this baseline be deleted as its outdated?

Reply:
Yep, oops. Didn't see CI!